CHANGELOG.md

```diff
@@ -1,3 +1,130 @@
+- CoreDNS version now based on Kubernetes version [#6501](https://github.com/kubermatic/kubermatic/issues/6501) 
+- Add support for "use-octavia" setting in Openstack provider specs. It defaults to "true" but leaves the possibility to set it to "false" if your provider doesn't support Octavia yet but Neutron LBaaSv2
+ [#6529](https://github.com/kubermatic/kubermatic/issues/6529) 
+- Add components override field to set nodeportrange for apiserver [#6533](https://github.com/kubermatic/kubermatic/issues/6533) 
+- Extend Kubermatic Constraint Template spec with a selector which enables cluster filtering for EE Kubermatic.  [#6534](https://github.com/kubermatic/kubermatic/issues/6534) 
+- Overhaul CA handling, allow to configure a global CA bundle for every component. The OIDC CA ile has been removed, manual updates can be necessary. [#6538](https://github.com/kubermatic/kubermatic/issues/6538) 
+- OpenShift support is removed.  [#6539](https://github.com/kubermatic/kubermatic/issues/6539) 
+- Fix issue with gatekeeper not recognizing the AdmissionReview v1 version by changing the webhook to use v1beta1 [#6550](https://github.com/kubermatic/kubermatic/issues/6550) 
+- OpenStack: Add support for "use-octavia" setting in Cluster Openstack cloud specs [#6565](https://github.com/kubermatic/kubermatic/issues/6565) 
+- Kubermatic Constraint Templates now support cluster filtering to allow users to manage the CT's and policies across different clusters more easily. The feature is available for Kubermatic EE.  [#6567](https://github.com/kubermatic/kubermatic/issues/6567) 
+- Fix KAS service port in Tunneling agent configuration. [#6569](https://github.com/kubermatic/kubermatic/issues/6569) 
+- Add support for Hetzner CCM [#6588](https://github.com/kubermatic/kubermatic/issues/6588) 
+- Increase default gatekeeper webhook timeout from 2 to 10 seconds, and added option in cluster settings to configure it. [#6590](https://github.com/kubermatic/kubermatic/issues/6590) 
+- Add support in Openstack datacenters to explicitly enable certain flavor types. [#6612](https://github.com/kubermatic/kubermatic/issues/6612) 
+- Fix the operator failing to reconcile the ValidatingWebhookConfiguration object for the cluster validation webhook [#6639](https://github.com/kubermatic/kubermatic/issues/6639) 
+- Provide the possibility of configuring leader election parameters for user cluster components. [#6641](https://github.com/kubermatic/kubermatic/issues/6641) 
+- Fix nodeport-proxy role used with LoadBalancer expose strategy. [#6646](https://github.com/kubermatic/kubermatic/issues/6646) 
+- Remove unused deprecated `certs` chart [#6656](https://github.com/kubermatic/kubermatic/issues/6656) 
+- Add `registry_mirrors` to Seed node settings [#6667](https://github.com/kubermatic/kubermatic/issues/6667) 
+- Make the `externalCloudProvider` feature gate impossible to disable once it has been enabled [#6668](https://github.com/kubermatic/kubermatic/issues/6668) 
```

Comment:
@xmudrii Should I separate this change in `# Cloud Provider` and if yes what other changes should I include.

Reply:
@imharshita This change is related to the CCM/CSI migration, which we removed from the changelog, so I think we should remove this entry.

Reply:
ok, will remove this one. what about others?

Reply:
I don't see other CCM/CSI migration-related changes besides this one.